com.github.alainm23.byte.json

```diff
@@ -0,0 +1,76 @@
+{
+    "app-id": "com.github.alainm23.byte",
+    "base": "io.elementary.BaseApp",
+    "base-version": "juno-19.08",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.alainm23.byte",
+    "finish-args": [
+        /* X11 + XShm */
+        "--share=ipc", "--socket=x11",
+        /* Wayland */
+        "--socket=wayland",
+        /* Filesystem */
+        "--filesystem=host",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        /* Network (Radio + Covers) */
+        "--share=network",
+        /* dconf */
+        "--metadata=X-DConf=migrate-path=/com/github/alainm23/byte/",
+        /* Audio */
+        "--socket=pulseaudio",
+        /* Notifications */
+        "--talk-name=org.freedesktop.Notifications",
+        /* Inhibitor */
+        "--talk-name=org.freedesktop.ScreenSaver",
+        /* Media Keys */
+        "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
+        "--talk-name=org.mpris.MediaPlayer2"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "taglib",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "sources": [{
+                "type": "archive",
+                "url": "https://github.com/taglib/taglib/archive/v1.11.1.tar.gz",
+                "sha256": "b6d1a5a610aae6ff39d93de5efd0fdc787aa9e9dc1e7026fa4c961b26563526b"
+            }]
+        },
+        {
+            "name": "byte",
+            "buildsystem": "meson",
+            "build-options": {
+                "config-opts": [
+                    "--buildtype=release"
+                ]
+            },
+            "sources": [{
+                "type": "archive",
+                "url": "https://github.com/alainm23/byte/archive/0.1.9.tar.gz",
+                "sha256": "cc0983f92340576ff7c2e3b434db65ef6a09f008975c4d7e9580427d3e19d6dd"
+            },{
+                "type": "patch",
+                "path": "elementary-theme.patch"
```

Comment:
You are upstream so either you want this upstream or you don't, it doesn't make sense to patch your own software.